src/locales/it-IT/admin.ftl

```diff
@@ -0,0 +1,1512 @@
+
+### Recent Comment History
+
+configure-moderation-recentCommentHistory-title = Cronologia recente
+configure-moderation-recentCommentHistory-timeFrame = Periodo di tempo della cronologia dei commenti recenti
+configure-moderation-recentCommentHistory-timeFrame-description =
+  Quantità di tempo per calcolare il tasso di rifiuto di un commentatore.
+configure-moderation-recentCommentHistory-enabled = Filtro della cronologia recente
+configure-moderation-recentCommentHistory-enabled-description =
+  Impedisce ai recidivi di pubblicare commenti senza approvazione.
+  Quando il tasso di rifiuto di un commentatore è superiore alla soglia, i suoi
+  commenti vengono inviati a Pending per l'approvazione del moderatore. Questo non si
+  si applica ai commenti dello Staff.
+configure-moderation-recentCommentHistory-triggerRejectionRate = Soglia del tasso di rifiuto
+configure-moderation-recentCommentHistory-triggerRejectionRate-description =
+  Commenti respinti ÷ (commenti respinti + commenti pubblicati)
+  nell'arco di tempo di cui sopra, in percentuale. Non include
+  i commenti in sospeso per tossicità, spam o pre-moderazione.
+
+#### Pre-Moderation
+configure-moderation-preModeration-title = Pre-moderazione
+configure-moderation-preModeration-explanation =
+Quando la pre-moderazione è attiva, i commenti non saranno pubblicati a meno che
+  approvati da un moderatore.
+configure-moderation-preModeration-moderation =
+  Pre-modera tutti i commenti
+configure-moderation-preModeration-premodLinksEnable =
+  Pre-modera tutti i commenti che contengono link
+configure-moderation-specificSites = Siti specifici
+configure-moderation-allSites = Tutti i siti
+
+configure-moderation-apiKey = Chiavi API
+
+configure-moderation-akismet-title = Filtro di rilevamento dello spam
+configure-moderation-akismet-explanation =
+  Il filtro API di Akismet avverte gli utenti quando un commento è determinato come probabile
+  essere spam. I commenti che Akismet ritiene essere spam non saranno pubblicati
+  e vengono messi in coda in attesa di revisione da parte di un moderatore.
+  Se approvato da un moderatore, il commento sarà pubblicato.
+
+configure-moderation-premModeration-premodSuspectWordsEnable =
+  Pre-moderare tutti i commenti che contengono parole sospette
+configure-moderation-premModeration-premodSuspectWordsDescription =
+  Puoi visualizzare e modificare la tua lista di parole sospette<wordListLink>qui</wordListLink>
+
+#### Akismet
+configure-moderation-akismet-filter = Filtro di rilevamento dello spam
+configure-moderation-akismet-ipBased = Rilevamento dello spam basato su IP
+configure-moderation-akismet-accountNote =
+  Nota: è necessario aggiungere il/i tuo/i dominio/i attivo/i
+Nel tuo account Akismet: <externalLink>https://akismet.com/account/</externalLink>
+configure-moderation-akismet-siteURL = URL sito
+
+
+#### Perspective
+configure-moderation-perspective-title = Filtro per i commenti tossici
+configure-moderation-perspective-explanation =

```

Comment:
It looks like the rest of this file is in English. Did you mean to include more Italian translations here?